changelog/fragments/ansible-path-issue-fix.yaml

```diff
@@ -0,0 +1,4 @@
+entries:
+  - description: >
+      For Ansible operators, API endpoint has path component in it. If it has path then current flow is ignoring the path component.
+    kind: bugfix
```

Comment:
highlight this as a breaking change, migration instructions are to wait for the proper fix.

Reply:
Modified.